3p/integration.js

```diff
@@ -131,6 +132,21 @@ import {zergnet} from '../ads/zergnet';
 import {zucks} from '../ads/zucks';
 
 /**
+ * If true, then in experiment where the passing of context metadata
+ * has been moved from the iframe src hash to the iframe name attribute.
+ */
+const nameExpOn = isExperimentOn(window, 'move-context-to-name');
```

Comment:
the flag value is not available inside iframe. (I'm working on it right now #6684 ). To not be blocked by me, I suggest you read from fragment first, and then from name.